packages/parser/__test__/parseJavascript.test.ts

```diff
@@ -258,9 +258,61 @@ test('Props: gets correct name for a quoted property', () => {
   expect(arg.default).toBe('HELLO')
 })
 
+const sfc2: AstResult = getAST('objectPropsWithSpreadElement.vue')
+test('SpreadElement in props object, get props not in SpreadElement', () => {
+  const mockOnProp = jest.fn(() => { })
+  const options: ParserOptions = {
+    onProp: mockOnProp
+  }
+  const seen = new Seen()
+  parseJavascript(sfc2.jsAst as bt.File, seen, options)
+  const arg = mockOnProp.mock.calls[0][0]
+
+  expect(mockOnProp.mock.calls.length).toBe(3)
+  expect(arg as PropsResult).toEqual({
+    name: 'a',
+    type: 'String',
+    describe: []
+  })
+})
+
+test('SpreadElement in props object, get props in SpreadElement', () => {
+  const mockOnProp = jest.fn(() => { })
+  const options: ParserOptions = {
+    onProp: mockOnProp
+  }
+  const seen = new Seen()
+  parseJavascript(sfc2.jsAst as bt.File, seen, options)
+  const arg = mockOnProp.mock.calls[1][0]
+
+  expect(mockOnProp.mock.calls.length).toBe(3)
+  expect(arg as PropsResult).toEqual({
+    name: 'b',
+    type: 'Number',
+    describe: []
+  })
+})
+
+test('SpreadElement in props object, get props in recursive SpreadElement', () => {
```

Comment:
What if our spreadElement has more than 2 nested level? Maybe 3, 4, 5 levels? Would you mind add  some test case for these scenerio?

Reply:
ofcourse.

Reply:
add test https://github.com/vuese/vuese/pull/262/commits/97fc78e3c941216e085900813378eeab5f441c1f